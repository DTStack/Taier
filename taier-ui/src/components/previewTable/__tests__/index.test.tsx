import { render } from '@testing-library/react';
import PreviewTable from '..';

describe('Test PreviewTable Component', () => {
    it('Should match snapshot', () => {
        const { asFragment } = render(
            <PreviewTable
                data={{
                    columnList: ['age', 'name'],
                    dataList: [
                        ['1', 'test'],
                        ['2', 'jest'],
                    ],
                }}
            />
        );

        expect(asFragment()).toMatchSnapshot();
    });
});
