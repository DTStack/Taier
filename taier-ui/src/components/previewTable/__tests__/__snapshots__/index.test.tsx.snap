// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test PreviewTable Component Should match snapshot 1`] = `
<DocumentFragment>
  <div
    class="p-4 h-full"
  >
    <div
      class="ant-table-wrapper taier__preview__table h-full"
    >
      <div
        class="ant-spin-nested-loading"
      >
        <div
          class="ant-spin-container"
        >
          <div
            class="ant-table ant-table-bordered"
          >
            <div
              class="ant-table-container"
            >
              <div
                class="ant-table-content"
              >
                <table
                  style="table-layout: auto;"
                >
                  <colgroup>
                    <col
                      style="width: 50px;"
                    />
                    <col
                      style="width: 60px;"
                    />
                  </colgroup>
                  <thead
                    class="ant-table-thead"
                  >
                    <tr>
                      <th
                        class="ant-table-cell"
                      >
                        age
                      </th>
                      <th
                        class="ant-table-cell"
                      >
                        name
                      </th>
                    </tr>
                  </thead>
                  <tbody
                    class="ant-table-tbody"
                  >
                    <tr
                      class="ant-table-row ant-table-row-level-0"
                      data-row-key="0"
                    >
                      <td
                        class="ant-table-cell"
                      >
                        1
                      </td>
                      <td
                        class="ant-table-cell"
                      >
                        test
                      </td>
                    </tr>
                    <tr
                      class="ant-table-row ant-table-row-level-0"
                      data-row-key="1"
                    >
                      <td
                        class="ant-table-cell"
                      >
                        2
                      </td>
                      <td
                        class="ant-table-cell"
                      >
                        jest
                      </td>
                    </tr>
                  </tbody>
                </table>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
